lib/tag-field/style.scss

```diff
@@ -9,9 +9,9 @@
   flex: 1 1 auto;
   line-height: 1.75em;
   white-space: nowrap;
-  overflow: visible;
-  max-height: 120px;
-	padding: 8px 12px;
+  overflow: auto;
+  max-height: 78px; // about 2.5 rows
```

Comment:
Not sure if this is better, but you could do something like `calc(2.5 * 1.75em + 16px);` to get the exact 'two and a half lines' value. (`1.75em` is the line height and `16px` of top and bottom padding)

Reply:
It'll also be better for when we `em/rem`-ify more `font-size` values for accessibility!